<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window</class>
 <widget class="QWidget" name="Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>80</width>
     <height>220</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AR PL UKai CN</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>实时数据</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>数据报表</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>历史曲线</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>供热指数</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>历史查询</string>
    </property>
   </item>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>240</width>
     <height>190</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>240</width>
       <height>190</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bitstream Charter</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/image/image/qixiangzhan.jpg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLCDNumber" name="fengliNumber">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>64</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value" stdset="0">
      <double>0.000000000000000</double>
     </property>
     <property name="intValue" stdset="0">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLCDNumber" name="guangzhaoNumber">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>110</y>
       <width>64</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value" stdset="0">
      <double>0.000000000000000</double>
     </property>
    </widget>
    <widget class="QLCDNumber" name="wenduNumber">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>64</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value" stdset="0">
      <double>0.000000000000000</double>
     </property>
     <property name="intValue" stdset="0">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>风力（m/s）</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>温度(°C)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>60</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>光照强度</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>90</y>
       <width>62</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>(kLux)</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>240</width>
       <height>190</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QPushButton" name="wenduButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>69</y>
       <width>80</width>
       <height>110</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/image/image/wendu.jpg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="fengliButton">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>70</y>
       <width>80</width>
       <height>110</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/image/image/fengli.jpg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="guangzhaoButton">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>80</width>
       <height>110</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/image/image/guangzhao.jpg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QTableView" name="tableView_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>140</width>
       <height>91</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>211</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>点击查看供热曲线</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>0</y>
       <width>131</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>供热参考表</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_5">
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>100</width>
       <height>28</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>温度</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>光照强度</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>风力</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>供热指数</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBox_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>100</width>
       <height>28</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>00:00</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>04:00</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>08:00</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>12:00</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>16:00</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>20:00</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>请选择类型</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>请选择时间</string>
     </property>
    </widget>
    <widget class="QPushButton" name="searchButton">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>90</y>
       <width>70</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>查询</string>
     </property>
    </widget>
    <widget class="QLabel" name="searchlabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>140</y>
       <width>240</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>120</y>
       <width>70</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/image/image/chaxuntest.jpg);</string>
     </property>
     <property name="text">
      <string>查询结果：</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="outButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>210</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/image/image/tuichu.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/image/image/xiaotiankong.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="timelabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>listWidget</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
